dressForTheWeather/Models/ClothingItem.swift

```diff
@@ -7,11 +7,22 @@
 //
 
 import Foundation
+import UIKit
 
 class ClothingItem {
     var name: String
     var placement: [BodyPlacement]
     var tempRange: ClosedRange<Double>
+    private var fallbackImages: [UIImage?] = [UIImage(named: "sparkle1") ?? nil,
```

Comment:
I'm not sure this is the best way to deal with these optionals, but I did find some good info on why they _should_ be optionals, at least: https://github.com/mac-cain13/R.swift/issues/61

Reply:
Maybe try something like this instead?

```
var image: UIImage? {
        guard let theImage = UIImage(named: name) else { return fallbackImage }
        return theImage
    }
    
    private let imageNames = ["sparkle1", "sparkle2", "sparkle3"]
    
    private var fallbackImage: UIImage? {
        let imageName = imageNames.randomElement() ?? ""
        return UIImage(named: imageName)
    }
```

Reply:
yep, I like this a lot better. Applied in c4a2f8619cdbae70fe0ac925df7ef9ad0a78de70